src/System.Transactions.Local/src/System/Transactions/TransactionManager.cs

```diff
@@ -377,7 +364,7 @@ public static TimeSpan MaximumTimeout
 
                 if (!s_cachedMaxTimeout)
                 {
-                    lock (ClassSyncObject)
+                    lock (LazyInitializer.EnsureInitialized(ref s_classSyncObject))
```

Comment:
Oops. Sorry. Missed reverting this one occurrence in addition to the lines mentioned in [above](https://github.com/dotnet/corefx/pull/16909#discussion_r105321175) discussion. Reverted.